audio8/ctc.py

```diff
@@ -128,11 +128,17 @@ def score_hyp(s):
 
 
 def postproc_letters(sentence):
+    sentence = ''.join(sentence)
     sentence = sentence.replace(" ", "").replace("|", " ").strip()
     return sentence
 
+def postproc_bpe(sentence):
+    sentence = ' '.join(sentence)
+    sentence = sentence.replace("@@ ", "").strip()
```

Comment:
don't you still need to strip out the blank symbol '|' ?

Reply:
during preproc the `wrd2bpe` script was using ' ' for blanks, but we could change that if we wanted.